apps/cow-amm-deployer/src/contexts/ammDataContext.tsx

```diff
@@ -47,6 +54,19 @@ export const AmmDataContextProvider = ({
     fetchWalletTokenBalance,
   );
 
+  useEffect(() => {
+    if (isPonderAPIUpToDate) {
+      mutateAmm();
+      setIsTransactionAwaiting(false);
+    }
+  }, [isPonderAPIUpToDate]);
+
+  useEffect(() => {
+    if (!["final", "confirmed", "error", "idle"].includes(status || "")) {
```

Comment:
we should remove `confirmed` from this now right?

Reply:
We're using more the ponder update as a trigger, so I think that is ok to let this